fossilize_external_replayer_linux.hpp

```diff
@@ -463,7 +463,11 @@ void ExternalReplayer::Impl::start_replayer_process(const ExternalReplayer::Opti
 
 	std::string self_path;
 	if (!options.external_replayer_path)
+#ifdef __linux__
+		self_path = "/proc/self/exe";
```

Comment:
This is fine. Just wrap the branch in braces.

Reply:
Makes sense, will be fixed in next push.